api/grpcserver/grpcserver_test.go

```diff
@@ -118,22 +117,6 @@ var (
 	stateRoot = types.HexToHash32("11111")
 )
 
-func dialGrpc(t testing.TB, cfg config.Config) *grpc.ClientConn {
```

Comment:
this is new code from @poszu where he factored out the duplicate code.
can you keep it? it doesn't seem to interfere with your change.

Reply:
@fasmat I don't understand why you basically reverted it. IMO, it's worth having this helper as it hides details unrelated to the test meat and makes sure that grpc connection handling is correct in every test. Duplicating it yields no benefit. 

You can still pass context and use `grpc.DialContext` here.

Reply:
Originally I had two `dialGrpc` functions, one with `context` and one without. That didn't feel right to me, since the function is just calling one other function (`grpc.DialContext`) and asserting that this function didn't return an error.

I can revert the change, although I'm generally opposed to `util` and `helper` methods 🙂